conda_build/cli/main_build.py

```diff
@@ -535,21 +535,20 @@ def execute(args: Sequence[str] | None = None):
 
     if "purge" in parsed.recipe:
         build.clean_build(config)
-        return
+        return 0
 
     if "purge-all" in parsed.recipe:
         build.clean_build(config)
         config.clean_pkgs()
-        return
+        return 0
 
-    outputs = None
     if parsed.output:
         config.verbose = False
         config.quiet = True
         config.debug = False
-        outputs = [output_action(recipe, config) for recipe in parsed.recipe]
+        for recipe in parsed.recipe:
+            output_action(recipe, config)
     elif parsed.test:
```

Comment:
We could also insert an `return 0` here, turn it into an `if parsed.test` and remove the `not parsed.output` in the last `if` at the bottom of the function.

Reply:
See https://github.com/conda/conda-build/pull/5169/commits/329bdafb422509452a0363da2422709482a08b9d